Tools/parametric_model/src/models/aerodynamic_models/linear_wing_model.py

```diff
@@ -0,0 +1,141 @@
+"""
+ *
+ * Copyright (c) 2021 Julius Schlapbach
+ *               2021 Autonomous Systems Lab ETH Zurich
+ * All rights reserved.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name Data Driven Dynamics nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+"""
+
+__author__ = "Julius Schlapbach"
+__maintainer__ = "Julius Schlapbach"
+__license__ = "BSD 3"
+
+import math
+import numpy as np
+
+from scipy.spatial.transform import Rotation
+from progress.bar import Bar
+
+
+class LinearWingModel():
+    def __init__(self, config_dict, mass):
```

Comment:
I don't think the linear wing model needs the mass information of the vehicle

Reply:
If the drag coefficient estimation should be done in the same way as before (in the `longitudinal-identification` repository), the mass is required to extract the measured lift coefficient from the available quantities in the dataset (see commented code in lines 80ff). Alternatively, it would only be possible to reparametrize the identification and to model drag as an e.g. quadratic function of the angle of attack.

Reply:
yes, has been removed right now with some code clean up